README.md

```diff
@@ -10,7 +10,7 @@ A collection of GCP resources ready to be used with [Humanitec](https://humanite
 
 The following resources are included:
 
-* None
+* [memorystore/redis](./humanitec-resource-defs/memorystore/redis): GCP Memorystore Redis resource.
```

Comment:
I wonder if we could move this to `./humanitec-resource-defs/redis/basic` as we generally have the Humanitec Resource Type as first element and the flavour second.